odxtools/compumethods/linearcompumethod.py

```diff
@@ -99,8 +99,10 @@ def convert_to_limit_to_physical(limit: Limit, is_upper_limit: bool) -> Limit:
             odxassert(isinstance(limit.value, (int, float)))
             if limit.interval_type == IntervalType.INFINITE:
                 return limit
-            elif (limit.interval_type == limit.interval_type.OPEN and
-                  self.internal_type.as_python_type() == int):
+            elif (limit.interval_type.value == limit.interval_type.OPEN and
+                  isinstance(self.internal_type.python_type, int)):
```

Comment:
looks odd here, isinstance used on type?

Reply:
I was surprised as well, but mypy says#
```
odxtools/compumethods/linearcompumethod.py:102: error: Non-overlapping equality check (left operand type: "Literal[IntervalType.OPEN, IntervalType.CLOSED]", right operand type: "Literal['OPEN']")  [comparison-overlap]
```

i.e., open intervals probably did not work properly before this PR...